Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -51,14 +51,20 @@ def __init__(self, *args, **kwargs):
 
 		self.ENABLE_FWU           = 1
 
+		# To enable source debug, set 1 to self.ENABLE_SOURCE_DEBUG
+		# self.ENABLE_SOURCE_DEBUG  = 1
+
+		# For test purpose
+		# self.SKIP_STAGE1A_SOURCE_DEBUG = 1
+
 		# OS_PK | FWU_PK | CFG_PK | FWU_PLD | PLD | Stage2 | Stage1B
 		self.VERIFIED_BOOT_HASH_MASK  = 0x000000D7
 		if self.ENABLE_FWU:
 			self.VERIFIED_BOOT_HASH_MASK  |= 0x00000028
 
 		self.STAGE1B_XIP          = 0
 
-		self.STAGE1A_SIZE         = 0x00008000
+		self.STAGE1A_SIZE         = 0x0000A000
```

Comment:
Why it is 0xa000 now?   APL SoC can only support up to 32KB IBBL (Stage1A).

Reply:
This is not for APL, but for QEMU to support Stage1A debugging. Let me increase the size depending on ENABLE_SOURCE_DEBUG.

Reply:
I'd prefer enabling debugging by default in QEMU so we cover some testing automatically.

Reply:
Oops, my mistake! 
But let us use ENABLE_SORUCE_DEBUG to control the size.  :)

Reply:
Aaron - In fact, QEMU can be debugged by gdb w/o DebugAgentLib. This is for testing purpose.